modules/config_games/server_configs/valheim_linux64.xml

```diff
@@ -0,0 +1,40 @@
+<game_config>
+ <game_key>valheim_linux</game_key>
+   <protocol>lgsl</protocol>
+  <lgsl_query_name>valheim</lgsl_query_name>
+ <installer>steamcmd</installer>
+ <game_name>Valheim</game_name>
+ <server_exec_name>valheim_server.x86_64</server_exec_name>
+ <cli_template>-name %HOSTNAME% %PORT% %HOME_PATH% %WORLD% %PASSWORD% -public= 1 </cli_template>
+ <cli_params>
+  <cli_param options="s" cli_string="-savedir" id="HOME_PATH"/>
+  <cli_param id="PORT" cli_string="-port "  />
```

Comment:
If the port line needs a space, better use the proper option instead of manually adding blank space here

Reply:
I removed in the blanks was my mistake